docker/Dockerfile.alpine.base

```diff
@@ -0,0 +1,14 @@
+ARG BASE_IMAGE=alpine:latest
+ARG CORES=${nproc}
+
+FROM ${BASE_IMAGE} as final
+
+RUN apk update
+
+RUN apk add --update alpine-sdk \
+    && apk add cmake openssl openssl-dev g++ \
+    curl-dev git autoconf libtool linux-headers
```

Comment:
should we add gtest/gmock too as part of this distro specific Dockerfile (and enable building tests). Also, how can we take care of thrift installation - should it be separate Dockerfile similar to way we are doing for  grpc?

Reply:
yes `gtest` and `gmock` could be part of the base. I would use a separate Dockerfile for `thrift` as well. This would allow users to reuse our Docker files in their CI.